### PR TITLE
[tests] run particular tests with bb

### PR DESCRIPTION
**Pull request by aldosolorzano**
# Problem
Desire to run a particular test ns

# Solution
Support command line args in the `bb test` command. Example:
```
bb test --focus runner.pipeline-test/pipeline-starts
```

https://github.com/bob-cd/bob/issues/123

### CODE REVIEW

File: runner/bb.edn [hunk 1]
```diff
@@ -31,7 +31,7 @@
                                     "artifact-store" "http://localhost:8001/ping"
                                     "postgres" "tcp://127.0.0.1:5433"
                                     "rabbitmq" "tcp://127.0.0.1:5673"})
-                       (shell "clojure -M:test")
+                       (apply shell "clojure -M:test" *command-line-args*)
```
Comment (aldosolorzano) [inline runner/bb.edn:34]: I only added it in the `runner` to ask for your input, but we could add it to each service.

Comment (lispyclouds) [reply runner/bb.edn:34]: Let's add it to both, keep it consistent.

Comment (lispyclouds) [reply runner/bb.edn:34]: Also would be nice to doc it in doc/dev.md

Comment (aldosolorzano) [reply runner/bb.edn:34]: will do

Comment (aldosolorzano) [reply runner/bb.edn:34]: @lispyclouds added the changes

**Pull request merged**